src/prototypes/draft-email-textarea/demo/demo.scss

```diff
@@ -0,0 +1,142 @@
+@use '../../../compiled/tokens/scss/size';
+@use '../../../compiled/tokens/scss/color';
+@use '../../../compiled/tokens/scss/ease';
+@use '../../../compiled/tokens/scss/transition';
+@use '../../../compiled/tokens/scss/z-index';
+@use 'sass:color' as sass-color;
+@use '../../../mixins/ms';
+
+///
+/// Tooltip styles
+///
+/// These Tooltip styles will be inlined in a Gutenberg Custom HTML block
+///
+/// TODO Consider making a Tooltip patterns component in the future
+///
+
+$tooltip-color-background: var(--theme-color-background-secondary);
+$tooltip-color-border: #{sass-color.change(
+    color.$brand-primary-darker,
+    $alpha: 0.4
+  )};
+
+///
+/// 1. Horizontally centers tooltip
+/// 2. Positions tooltip vertically (below button)
+///
+
+._c-tooltip {
```

Comment:
Nit: I'm not used to this class syntax. Is it just because this is "prototype" code and not a real component? (If so, that's fine. I just wanted to make sure this was intentional)

Reply:
@Paul-Hebert Yes, indeed! 

Props to @tylersticka for suggesting this [underscore prefix formatting](https://github.com/cloudfour/cloudfour.com-patterns/pull/2109#discussion_r1065157734).